tf/terraform.go

```diff
@@ -86,21 +82,27 @@ func ParseModules(path string) ([]Module, error) {
 		if len(block.Labels) == 1 {
 			moduleName = block.Labels[0]
 		} else {
-			errs.Append(errors.E(ErrTerraformSchema, block.OpenBraceRange,
-				"\"module\" block must have 1 label"))
+			logger.Debug().Msgf("ignoring module block with %d labels", len(block.Labels))
+
+			continue
 		}
+		logger = logger.With().
+			Str("module", moduleName).
+			Logger()
 
 		logger.Trace().Msg("Get source attribute.")
 		source, ok, err := findStringAttr(block, "source")
 		if err != nil {
-			errs.Append(errors.E(ErrTerraformSchema, err,
-				"looking for module.%q.source attribute", moduleName))
+			logger.Debug().
+				Err(err).
+				Msg("ignoring module block without source")
```

Comment:
in this case, it's not a missing source but it failed to evaluate the source expression.